e2e/service_cosmos_test.go

```diff
@@ -3,51 +3,162 @@
+		case sharedtypes.RPCType_REST:
+			cosmosSDKMethods := getCosmosSDKRESTTestURLPaths()
 
-// Reference for all CometBFT RPC URL paths:
+			cosmosSDKTargets, err := getCosmosSDKRESTVegetaTargets(ts, cosmosSDKMethods, gatewayURL)
+			if err != nil {
+				return nil, err
+			}
+			maps.Copy(targets, cosmosSDKTargets)
+
+		case sharedtypes.RPCType_JSON_RPC:
+			// For EVM JSON-RPC, we use the same targets as the EVM service.
+			evmJSONRPCTargets, err := getEVMVegetaTargets(ts, gatewayURL)
+			if err != nil {
+				return nil, err
+			}
+			maps.Copy(targets, evmJSONRPCTargets)
+		}
+	}
+
+	return targets, nil
+}
+
+/* -------------------- CometBFT JSON-RPC Method Definitions -------------------- */
+
+var cosmosSDKExpectedID = jsonrpc.IDFromInt(1)
+
+// Reference for all CometBFT RPC methods:
 // - https://docs.cometbft.com/v0.38/rpc/
 const (
-	cometbftEndpointStatus         = "/status"          // Get node status
-	cometbftEndpointHealth         = "/health"          // Get node health
-	cometbftEndpointNetInfo        = "/net_info"        // Get network info
-	cometbftEndpointConsensusState = "/consensus_state" // Get consensus state
-	cometbftEndpointCommit         = "/commit"          // Get commit
-	cometbftEndpointABCIInfo       = "/abci_info"       // Get ABCI info
-	cometbftEndpointBlock          = "/block"           // Get block at height
-	cometbftEndpointBlockResults   = "/block_results"   // Get block results
-	cometbftEndpointValidators     = "/validators"      // Get validators
+	cometbftMethodStatus         = "status"          // Get node status
+	cometbftMethodHealth         = "health"          // Get node health
+	cometbftMethodNetInfo        = "net_info"        // Get network info
+	cometbftMethodConsensusState = "consensus_state" // Get consensus state
+	cometbftMethodCommit         = "commit"          // Get commit
+	cometbftMethodABCIInfo       = "abci_info"       // Get ABCI info
+	cometbftMethodBlock          = "block"           // Get block at height
+	cometbftMethodBlockResults   = "block_results"   // Get block results
+	cometbftMethodValidators     = "validators"      // Get validators
 )
 
-// getCometBFTTestURLPaths returns all CometBFT URL paths for a service load test.
-func getCometBFTTestURLPaths() []string {
+// getCometBFTTestMethods returns all CometBFT JSON-RPC methods for a service load test.
+func getCometBFTTestMethods() []string {
 	return []string{
-		cometbftEndpointStatus,
-		cometbftEndpointHealth,
-		cometbftEndpointNetInfo,
-		cometbftEndpointConsensusState,
-		cometbftEndpointCommit,
-		cometbftEndpointABCIInfo,
-		cometbftEndpointBlock,
-		cometbftEndpointBlockResults,
-		cometbftEndpointValidators,
+		cometbftMethodStatus,
+		cometbftMethodHealth,
+		cometbftMethodNetInfo,
+		cometbftMethodConsensusState,
+		cometbftMethodCommit,
+		cometbftMethodABCIInfo,
+		cometbftMethodBlock,
+		cometbftMethodBlockResults,
+		cometbftMethodValidators,
 	}
 }
 
 func getCometBFTVegetaTargets(
```

Comment:
Can you see a way to decouple CometBFT and CosmosSDK?

The reason I ask is beacuse I'm seeing lots of teams who use CometBFT (and not CosmosSDK), so it'll make it easier for them to adopt/test PATH when the time comes.